docs/experiment_pv_data.md

```diff
@@ -0,0 +1,135 @@
+    - [Response 200 OK](#response-200-ok)
+  - [Get experiment data](#get-experiment-data)
+    - [Response 200 OK](#response-200-ok-1)
+    - [Response 404 NOT_FOUND - User in experiment not found](#response-404-NOT_FOUND---user-in-experiment-not-found)
+
+
+## Database models
+
+### ProcessVariable
+
+| Column name         | type                              | primary key | unique | nullable | default value |
+|---------------------|-----------------------------------|-------------|--------|----------|---------------|
+| pv_string           | String(100)                       | True        | True   | False    | ---           |
+| human_readable_name | String(100)                       | False       | False  | True     | ---           |
+| experiment_short_id | ForeignKey("experiment.short_id") | False       | False  | False    | ---           |
+
+*The `short_id` is automatically taken from the `pv_string`. It consists of all the characters before the first `:`.*
+
+### Experiment
+
+| Column name         | type                            | primary key | unique | nullable | default value |
+|---------------------|---------------------------------|-------------|--------|----------|---------------|
+| short_id            | db.String(100)                  | True        | True   | False    | ---           |
+| human_readable_name | db.String(100)                  | False       | False  | True     | ---           |
+| process_variables   | relationship("ProcessVariable") | False       | False  | True     | ---           |
+| users               | db.relationship("User")         | False       | False  | True     | ---           |
+
+### User
+
+| Column name        | type                      | primary key | unique | nullable | default value |
+|--------------------|---------------------------|-------------|--------|----------|---------------|
+| user_id            | Text(36)                  | True        | True   | False    | uuid4 or int  |
+| login_name         | String(100)               | False       | True   | False    | ---           |
+| first_name         | String(100)               | False       | False  | False    | ---           |
+| email              | String(100)               | False       | False  | True     | ---           |
+| password_hash      | String(100)               | False       | False  | False    | ---           |
+| user_type          | UserTypeEnum              | False       | False  | False    | "User"        |
+| experiments        | db.backref("experiments") | False       | False  | False    | ---           |
+| preferred_language | String(10)                | False       | False  | False    | "en"          |
+
+*In the database there is a table users_to_experiments, which handles the relationship between users and experiments.*
+
+## Endpoints
+
+### Get a list of experiments
+
+- Endpoint: `experiments/data`
+- Method: `POST`
+
+*Send an **admin** access token in the "x-access-tokens" header!*
```

Comment:
Not admins! All Users in the Experiment have access.

Reply:
Done!